client/modules/ftml-components/src/components/footnotes/_footnotes.scss

```diff
@@ -1,8 +1,78 @@
-a.wj-footnote-ref {
-  display: inline-block;
+.wj-footnote-list-item-marker,
+.wj-footnote-ref-marker {
+  @include reset;
+  display: inline;
+  color: #0000ee;
```

Comment:
I copied the color straight from [standardized HTML styling recommendations](https://html.spec.whatwg.org/multipage/rendering.html#phrasing-content-3). It could be shortened, but there is an actual meaning behind me picking this _exact_ color.

Reply:
Huh, okay